src/lang/modifyAst/addShell.ts

```diff
@@ -92,6 +102,8 @@ export function addShell({
     pathToExtrudeNode,
     'VariableDeclarator'
   )
+  console.log('pathToExtrudeNode', pathToExtrudeNode)
+  console.log('extrudeNode', extrudeNode)
```

Comment:
```suggestion
```
[nit] could probably remove unless you want these in for a short while for production visibility?

Reply:
Thanks Frank!